modules/contrib/webform/webform.module

```diff
@@ -636,7 +636,7 @@ function webform_submission_access($node, $submission, $op = 'view', $account =
 
   // If access is granted via a token, then allow subsequent submission access
   // for anonymous users.
-  if (!$user->uid && $token_access) {
+  if (!$account->uid && $token_access) {
```

Comment:
Note: I'm concerned that this line may affect our complicated webform submission access rules for webform submission access.

Reply:
With how `$account` is set on line 629 (using `$user` as a default), I think this will be okay.